agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/Indexer.java

```diff
@@ -212,6 +213,7 @@ public void run() {
 	protected abstract void startSingleThread(LinkedBlockingDeque<String> queue);
 
 	protected ObjectMapper customizeObjectMapper(ObjectMapper objectMapper) {
+		objectMapper.registerModule(new JavaTimeModule());
```

Comment:
@markquintontulloch I didn't catch this... this effects ALL indexers. I think what needs to happen like in the disease indexer there is an override:
```java
	@Override
	protected ObjectMapper customizeObjectMapper(ObjectMapper objectMapper) {
		return RestConfig.config.getJacksonObjectMapperFactory().createObjectMapper();
	}
```